docs/reference/openapi-rules.md

```diff
@@ -658,3 +727,122 @@ Validate structure of OpenAPI v3 specification.
 Parameter objects should have a `description`.
 
 **Recommended:** No
+
+### oas3-valid-media-example
+
+Examples must be valid against their defined schema. This rule is applied to Media objects, such as requests and responses.
```

Comment:
request and responses aren't media objects though.
Media Type Object is just a standalone object https://spec.openapis.org/oas/v3.1.0#media-type-object that may be present under "content" in requests or responses

Reply:
I think this is correct in the sense that a media type object is used in a request and a response. Requests and responses both have media types in there somewhere.